docs/README.md

```diff
@@ -31,13 +31,25 @@ docker run -it -d --name odtn-emulator_tapi_ols_1 -p 11000:1234 onosproject/tapi
 ```
 
 ## Test if the Cassini docker runs sucessfully
-If you want to check whether the containers start up, there are three commands to test three containers respectively:
+If you want to check whether the containers start up, you can use *netconf-console* and *curl*. To install *netconf-console* which is a  console application for interacting with NETCONF servers built on top of ncclient as follows.
+
+```bash
+pip3 install netconf-console
+```
+
+**Note**: You also need to make sure   ncclient version **0.6.3**  is installed on your machine. If higher versions of ncclient installed on your machine, you need to reinstall it as follows:
```

Comment:
can you please ad the command used to check the version of ncclient please ?

Reply:
Sure.